src/core-class-example.js

```diff
@@ -1,11 +1,4 @@
 class Module {
-  constructor(command) {
-    if (!command || typeof command !== 'string') {
-      throw Error('Command must be a string.');
-    }
-    this.enabled = true;
-  }
-
   onReceive(sender) {
     console.log('onReceive');
```

Comment:
Maybe we shouldn't add `sender` as a parameter of every method.
Maybe we can add it as an injected field, what do you think?